### DISCUSSION TITLE
Deep Phonemizer

### DISCUSSION THREAD

**Discussion by georgedei** [yl4579/StyleTTS2#158] [Q&A]
Transferred the codebase from colab and got it working on my local machine. At least the inference in English. However the Phonemizer produces a lot of garbled phonemes. I am using the dpphonemizer and have the en_us_cmudict_ipa_forward model installed. What steps could I take for troubleshooting? Or try another phonemizer?
Thanks.

**Comment by yl4579** [comment]
The pre-trained model only works with phonemizer (espeak backend) for now. If you want to use a different phonemizer, you have to re-train the model from scratch.
